FishingAgency/Controller/DeleteShipController.cs

```diff
@@ -12,7 +12,7 @@ public class DeleteShipController
     {
         public void DeleteShip(string shipName)
         {
-            using (FishingAgencyEntities fadb = new FishingAgencyEntities())
+            using (FishingAgencyEntities2 fadb = new FishingAgencyEntities2())
```

Comment:
I disagree "FishingAgencyEntities2 fadb = new FishingAgencyEntities2()" to be that way because on the other PC's is working fine

also
![image](https://user-images.githubusercontent.com/57759807/172692180-f04293f5-13f5-4d78-9221-0148782caea6.png)